_data/cv.yml

```diff
@@ -0,0 +1,97 @@
+- title: General Information
+  type: map
+  contents:
+    - name: Full Name
+      value: Albert Einstein
+    - name: Date of Birth
+      value: 14th March 1879
+    - name: Languages
+      value: English, German
+
+- title: Education
+  type: table
```

Comment:
could we change the name of this type to `time_table`? this would indicate that time information (the `year` field) has some special treatment.